src/workerd/io/actor-sqlite.c++

```diff
@@ -662,6 +662,45 @@ kj::Maybe<kj::Promise<void>> ActorSqlite::onNoPendingFlush() {
   return outputGate.wait();
 }
 
+kj::Promise<kj::String> ActorSqlite::getCurrentBookmark() {
+  // This is an ersatz implementation that's good enough for local dev with D1's Session API.
```

Comment:
I had to look up the word "ersatz". I had assumed it was the name of some algorithm or something, but apparently it just means "fake". Maybe the word "fake" would be more approachable?

Reply:
I decided to call a spade a spade and use localDevelopment instead of ersatz.

Reply:
"ersatz" is such a beautiful word tho. I mourn its loss here. But yeah, probably one that's best to avoid lol.